source/deprecations/v1.x.html.md

```diff
@@ -1022,4 +1022,23 @@ c = Contact.create({ firstName: "John", lastName: "Doe" });
 c.swapNames();  // returns c
 Object.freeze(c);
 c.swapNames();  // EXCEPTION
-```
\ No newline at end of file
+```
+
+
+#### CreateWithMixins
+
+`Ember.createWithMixins` method has been deprecated in favor of create or extend.
```

Comment:
Ember.Object.createWithMixins (not Ember.createWithMixins).

The replacement is to call Ember.Object.extend().create() (so extend AND create).